get_data.py

```diff
@@ -0,0 +1,110 @@
+"""
+Download all required data for training and / or testing of model for classifying food images
+
+Setting 'mode' flag to train will download:
+- Resized images for relevant food classes (all as 256 x 256 pixels)
+- Pre-trained Alexnet model weights
+
+Setting 'mode' flag to test will download:
+- Resized images for relevant food classes (all as 256 x 256 pixels)
+- Fine-tuned Alexnet model weights
+- Mean image binaryproto used during training
+
+"""
+
+import os
+import sys
+import argparse
+import urllib
+import time
+import hashlib
+import tarfile
+
+parser = argparse.ArgumentParser()
```

Comment:
This should be in the `if __name__ == "__main__":` block. Or in the separate function.